app/client/utils.js

```diff
@@ -1,168 +1,203 @@
+
+export function formatMoney(
+    amount,
+    decimalCount = window.CURRENCY_DECIMALS,
+    decimal = ".",
+    thousands = ",",
+    currency = window.CURRENCY_NAME
+) {
+    try {
+        decimalCount = Math.abs(decimalCount);
+        decimalCount = isNaN(decimalCount) ? 2 : decimalCount;
+
+        const negativeSign = amount < 0 ? "-" : "";
+
+        let i = parseInt(
+            (amount = Math.abs(Number(amount) || 0).toFixed(decimalCount))
+        ).toString();
+        let j = i.length > 3 ? i.length % 3 : 0;
+
+        return (
+            negativeSign +
+            (j ? i.substr(0, j) + thousands : "") +
+            i.substr(j).replace(/(\d{3})(?=\d)/g, "$1" + thousands) +
+            (decimalCount
+                ? decimal +
+                  Math.abs(amount - i)
+                      .toFixed(decimalCount)
+                      .slice(2)
+                : "") +
+            " " +
+            currency
+        );
+    } catch (e) {
+        console.log(e);
+    }
 }
 
 const overwriteMerge = (destinationArray, sourceArray) => sourceArray;
 
 export function DEEEEEEP(parent_object, child_object_to_add) {
-  // update object state data with new data, while keeping untouched old data, overwrite array
-  return merge(parent_object, child_object_to_add, { arrayMerge: overwriteMerge })
+    // update object state data with new data, while keeping untouched old data, overwrite array
+    return merge(parent_object, child_object_to_add, {
+        arrayMerge: overwriteMerge
+    });
 }
 
-export function addCreditTransferIdsToTransferAccount(parent_object, child_object_to_add) {
-  // update object state data with new data, while keeping untouched old data, merge arrays
-  return merge(parent_object, child_object_to_add)
+export function addCreditTransferIdsToTransferAccount(
+    parent_object,
+    child_object_to_add
+) {
+    // update object state data with new data, while keeping untouched old data, merge arrays
+    return merge(parent_object, child_object_to_add);
 }
 
-export const generateQueryString = (query) => {
-  let query_string = '?';
+export const generateQueryString = query => {
+    let query_string = "?";
 
-  if (query) {
-    Object.keys(query).map(query_key => {
-      let string_term = query_key + '=' + query[query_key] + '&';
-      query_string = query_string.concat(string_term)
-    });
-  }
+    if (query) {
+        Object.keys(query).map(query_key => {
+            let string_term = query_key + "=" + query[query_key] + "&";
+            query_string = query_string.concat(string_term);
+        });
+    }
 
-  let orgId = getOrgId();
-  var response_string = query_string;
-  if (orgId !== null && typeof orgId !== "undefined") {
-    response_string = query_string + `org=${orgId}&`;
-  }
+    let orgId = getOrgId();
+    var response_string = query_string;
+    if (orgId !== null && typeof orgId !== "undefined") {
+        response_string = query_string + `org=${orgId}&`;
+    }
```

Comment:
defs need a linter

Reply:
haha..there were moments were I had to find a way to save without formatting because my VSCode setup has a few packages that default to format on save. I made a ticket for me (or anyone else) to visit!

https://trello.com/c/rFzwJZHg/257-setup-linter-to-standardize-code-styling

Reply:
awesome!